apps/docs/src/data/components/button.data.ts

```diff
@@ -9,64 +9,201 @@ export default {
         },
         {
           prop: 'pressed',
           type: 'Booleanish',
           default: undefined,
+          description: `When set to 'true', gives the button the appearance of being pressed and adds attribute 'aria-pressed="true"'. When set to 'false' adds attribute 'aria-pressed="false"'. Syncable with the v-model:pressed`,
         },
         {
           prop: 'size',
           type: 'Size',
           default: 'md',
+          description: `Set the size of the component's appearance`,
         },
         {
           prop: 'squared',
           type: 'Booleanish',
           default: false,
+          description: `Renders the button with non-rounded corners when set to 'true'`,
         },
         {
           prop: 'tag',
           type: 'string',
           default: 'button',
+          description: `Specify the HTML tag to render instead of the default tag`,
         },
         {
           prop: 'type',
           type: 'ButtonType',
           default: 'button',
+          description: `The value to set the button's 'type' attribute`,
         },
         {
           prop: 'variant',
           type: 'ButtonVariant | null',
           default: 'secondary',
+          description: `Applies one of the Bootstrap theme color variants to the component`,
         },
         {
           prop: 'loading',
           type: 'Booleanish',
           default: false,
+          description: `Determine whether it's loading, render the loading text & spinner`,
         },
         {
           prop: 'loadingFill',
           type: 'Booleanish',
           default: false,
+          description: `Only render spinner when loading`,
         },
         {
           prop: 'block',
           type: 'Booleanish',
           default: false,
+          description: `Renders a 100% width button (expands to the width of its parent container)`,
         },
         {
           prop: 'loadingText',
           type: 'string',
           default: 'Loading...',
+          description: `Loading text that displays to the left the spinner`,
+        },
+        {
+          prop: 'active',
+          type: 'Booleanish',
+          default: undefined,
+          description: `When set to 'true', places the component in the active state with active styling`,
+        },
+        {
+          prop: 'activeClass',
+          type: 'string',
+          default: undefined,
+          description: `<router-link> prop: Configure the active CSS class applied when the link is active`,
+        },
+        {
+          prop: 'exactActiveClass',
+          type: 'string',
+          default: undefined,
+          description: `<router-link> prop: Configure the active CSS class applied when the link is active with exact match`,
+        },
+        {
+          prop: 'append',
+          type: 'Booleanish',
+          default: undefined,
+          description: `<router-link> prop: Setting append prop always appends the relative path to the current path`,
```

Comment:
```suggestion
          description: `<RouterLink> prop: Setting append prop always appends the relative path to the current path`,
```
This suggestion goes for all components written in kebab case. **Everything** should be Pascal case

Reply:
Ok thanks